src/routes/donations.js

```diff
@@ -213,53 +213,60 @@ function _simpleCache(Model) {
     populate() {
       return Model.findAll().then((items) => {
         const obj = items.reduce((cache, model) => {
-          cache[model.description] = model.id
-          return cache
-        }, {})
-        this.data = obj
-        return this
-      })
+          cache[model.description] = model.id;
+          return cache;
+        }, {});
+        this.data = obj;
+        return this;
+      });
     },
     findOrCreate(description) {
-      if (!description) return Promise.resolve()
-      const found = this.data[description]
+      if (!description) return Promise.resolve();
+      const found = this.data[description];
       if (found) {
-        return Promise.resolve(found)
+        return Promise.resolve(found);
       }
       return Model.create({
         description,
       }).then((model) => {
-        this.data[description] = model.id
-        return model.id
-      })
+        this.data[description] = model.id;
+        return model.id;
+      });
     },
-  }
+  };
 }
 
 function _buildDonation(csvDonation) {
   return {
-    donationDate: csvDonation['Date of Donation'],
-    donationAmount: csvDonation['Amount'],
-    donationType: csvDonation['Type of Donation'],
-    remarks: csvDonation['Remarks'],
-    receiptNo: csvDonation['Receipt Serial No'],
-    void: csvDonation['Void'],
-    taxDeductible: csvDonation['Tax Deductible'] || false,
-  }
+    donationDate: csvDonation["Date of Donation"],
+    donationAmount: csvDonation["Amount"],
+    donationType: csvDonation["Type of Donation"],
+    remarks: csvDonation["Remarks"],
+    receiptNo: csvDonation["Receipt Serial No"],
+    void: csvDonation["Void"],
+    taxDeductible: convertToBoolean(csvDonation["Name of Fund"]),
+  };
 }
 
 function _buildDonor(csvDonation) {
   return {
-    idNo: csvDonation['ID No'] || null,
-    name: csvDonation['Name'],
-    email: csvDonation['Email Address'],
-    contactNo: csvDonation['Tel No'],
-    address1: csvDonation['Add 1'],
-    address2: csvDonation['Add 2'],
-    postalCode: csvDonation['Postal Code'],
-  }
+    idNo: csvDonation["ID No"] || null,
+    name: csvDonation["Name"],
+    email: csvDonation["Email Address"],
+    contactNo: csvDonation["Tel No"],
+    address1: csvDonation["Add 1"],
+    address2: csvDonation["Add 2"],
+    postalCode: csvDonation["Postal Code"],
+  };
 }
 
+function convertToBoolean(data) {
```

Comment:
No need to change it now, but a better name would be `isTaxDeductible` - it gives some clue as to what the resulting Boolean value means.